packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamTable.tsx

```diff
@@ -149,6 +180,7 @@ export default function ChallengeTeamTable(props: ChallengeTeamTableProps) {
         rowCount={5}
         hideFooter
         density="compact"
+        rowBuffer={5} // Buffer ensures all rows required in test are loaded.
```

Comment:
MUI DataGrid has virtualization enabled by default, which causes problems in JSDOM when it doesn't load all of the rows you expect to see. This prop ensures enough rows are loaded for our tests to work.